loader/loader.c

```diff
@@ -2631,9 +2630,13 @@ static VkResult loader_add_layer_properties(const struct loader_instance *inst,
             } while (layer_node != NULL);
         }
     }
+    
+    
 
 out:
-
+    if(NULL != file_vers) {
```

Comment:
Coding standrad:  You need a space after the if.  After that, I'll push it through our test environment.